index.js

```diff
@@ -67,11 +69,12 @@ function traceModuleFunction(module, emit) {
 
         const event = new Event(name);
         for (let i = 0; i !== numInputActions; i++) {
-          const item = inputActions[i];
-          const action = item[0];
-          const params = item[1];
+          const [ action, params ] = inputActions[i];
           action(values, event, params);
         }
+        if (isFunction(spec.callbacks.onEnter)) {
+          spec.callbacks.onEnter(event, this.context);
```

Comment:
for example you may want the backtrace only sometimes... but yeah - all the info are in the event already, so maybe it's better to leave it like this